doc/ref_model/chapters/chapter02.md

```diff
@@ -171,7 +171,7 @@ Based on the above analysis, following NFVI profiles are proposed (Also shown in
 
 - **Basic**: VNFCs that perform basic compute operations.
 - **Network Intensive**: VNFCs that perform network and compute intensive operations with high throughput and low latency requirements.
-- **Compute Intensive**: VNFCs that perform compute intensive operations.
+- **Compute Intensive**: VNFCs that perform compute intensive operations (but do not have high network throughput and low latency requirements).
 - **Storage Intensive**: VNFCs that perform storage intensive operations with high IOPS requirements. (_**Note:** Storage Intensive Profile will not be defined in initial CNTT releases_)
```

Comment:
@xavier-grall did you mean " but do not have high network throughput and *have* low latency requirements1"

Reply:
@ASawwaf  @pgoyal01
No, so the wording needs to be more precise: maybe  "but do not have **neither** high network throughput **nor** low **network** latency requirements"
But it is a proposal from Pankaj, so his opinion is needed.
In my view the low latency required for control functions targeted by that profile is provided at computation level by cpu pinning. But not so sure, and a minimal network acceleration may also be needed (eg, a software acceleration with very few dpdk pmd threads).

Reply:
@xavier-grall @pgoyal01  ," but do not have neither high network throughput nor low network latency requirements" this definition is valid for basic or general-purpose profiles 

But for Control plan VNF ( as IMS ) : I think it is required low latency which should be onboarded on Compute intensive profile 

what do u think ?

Reply:
In my opinion, low latency requirements of some control plane functions (as IMS CSCF, MME...) are addressed by providing these functions with sufficient, ie dedicated, NFVI computation resources, but not with specific NFVI networking resources. 
And these dedicated computation resources are reflected by the "fast computing" requirement of CI profile (eg, provided by cpu pinning)
But, as I also said previously, maybe I'm wrong and we should also consider some specific networking resources and an additional low latency requirement (eg provided by dpdk-based vswitching).
@pgoyal01, @kedmison, @ulikleber 
Thank you in advance for your opinion on that point.

Reply:
We are talking about generic profiles here. In reality many VNFs (better VNFCs) only roughly fall into one of 3 categories and have specific needs to reach their optimum effectivity.

Reply:
I would suggest wording: "Basic: VNFCs that perform basic compute operations and can tolerate oversubscription the variable compute latency."

Reply:
@ASawwaf I agree with @xavier-grall the control plane latency requirements will be addressed by CPU and memory.

Reply:
I suggest to deal with the evolution of profiles requirements and definitions from another issue to create.
And right now, to merge the current PR for taken into account:
- the initial issue by Rabi (CI/NI figures inverting) 
- the example vnf change by Ahmed (PGW on NI)
- the example vnf change by Kelvin (no PCRF on Basic).